.github/workflows/docs-update.yml

```diff
@@ -0,0 +1,36 @@
+name: Update OpenTelemetry Website Docs
+
+on:
+  # triggers only on a manual dispatch
+  workflow_dispatch:
+
+jobs:
+  update-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: make-pr
+        env:
+          API_TOKEN_GITHUB: ${{secrets.GITHUB_PAT_TOKEN}}
```

Comment:
for unknown reasons, it looks like this is not a valid secret name. How about `DOC_UPDATE_TOKEN` ?